src/core.js

```diff
@@ -425,6 +425,16 @@ jQuery.extend({
 	support: support
 });
 
+// jsHint would error on this code due to the Symbol not being defined in ES5.
+// Defining this global in .jshintrc would create a danger of using the global
+// unguarded in another place, it seems safer to just disable jsHint for these
+// three lines.
+/* jshint ignore: start */
+if ( typeof Symbol === "function" && Symbol.iterator ) {
```

Comment:
I'd be fine with removing the truthy `Symbol.iterator` guard completely, simplifying this condition to `if ( typeof Symbol === "function" )`. AFAICT, the worst outcome of such would be a useless property on the jQuery prototype.